src/allotropy/allotrope/models/shared/definitions/custom.py

```diff
@@ -1,3 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
NOTE: This file is now auto-generated and alphabetized, hence the large amount of diffs